### ISSUE TITLE
feat: Make validation timeout configurable

### ISSUE THREAD

**Issue by nepalevov** [terraform-aws-modules/terraform-aws-acm#123]
## Is your request related to a problem? Please describe.

Since aws provider **4.12.0** the default timeout for resource `aws_acm_certificate_validation` was increased from 45 to 75 minutes (https://github.com/hashicorp/terraform-provider-aws/issues/20073), so the \
I use this module in CI and new resource timeout exceeds default CI job timeout, which is 60 minutes.

## Describe the solution you'd like.

I'd like to have **create timeout** for resource `"aws_acm_certificate_validation" "this" {}` configurable by variable, e.g.

```hcl
variable "validation_timeout" {
  description = "Define maximum timeout to wait for the validation to complete"
  type        = string
  default     = "75m"
}

resource "aws_acm_certificate_validation" "this" {
  count = local.create_certificate && var.validation_method != "NONE" && var.validate_certificate && var.wait_for_validation ? 1 : 0

  certificate_arn = aws_acm_certificate.this[0].arn

  validation_record_fqdns = flatten([aws_route53_record.validation.*.fqdn, var.validation_record_fqdns])

  timeouts {
    create = var.validation_timeout
  }
}
```

## Describe alternatives you've considered.

Currently, I have to set `wait_for_validation = false`. I'd prefer to set smaller validation timeout and receive terraform error, though.

**Comment by antonbabenko** [comment]
Please open a PR for this change. The variable's default value should be `null` (not "75m") to respect Terraform AWS provider's setting.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.2.0 :tada:
